jetty-server/src/main/java/org/eclipse/jetty/server/SecureRequestCustomizer.java

```diff
@@ -363,6 +362,30 @@ public Object getAttribute(String name)
             return null;
         }
 
+        /**
+         * Get data belonging to the {@link SSLSession}.
+         *
+         * @return the SslSessionData
+         */
+        private SslSessionData getSslSessionData()
+        {
+            SslSessionData sslSessionData = (SslSessionData)_session.getValue(SslSessionData.ATTR);
+            if (sslSessionData == null)
+            {
+                String cipherSuite = _session.getCipherSuite();
+                int keySize = SslContextFactory.deduceKeyLength(cipherSuite);
+
+                X509Certificate[] certs = getCertChain(_request.getHttpChannel().getConnector(), _session);
+
+                byte[] bytes = _session.getId();
+                String idStr = TypeUtil.toHexString(bytes);
+
+                sslSessionData = new SslSessionData(keySize, certs, idStr);
+                _session.putValue(SslSessionData.ATTR, sslSessionData);
+            }
+            return sslSessionData;
```

Comment:
This method is the only user of `SslSessionData.ATTR`.
So perhaps we don't need the constant in another class.
We can do at beginning of method:

`String key = SslSessionData.class.getName()`

and then use `key` where now we use `ATTR`.

Reply:
Done.